axelrod/strategies/meta.py

```diff
@@ -533,3 +535,83 @@ def __init__(self):
                 <= 1]
         super().__init__(team=team)
         self.classifier["long_run_time"] = False
+
+class MemoryDecay(MetaPlayer):
+    """
+    A player utilizes the (default) Tit for Tat strategy for the first (default) 15 turns,
+    at the same time memorizing the opponent's decisions. After the 15 turns have
+    passed, the player calculates a 'net cooperation score' (NCS) for his opponent,
+    weighing decisions to Cooperate as (default) 1, and to Defect as (default)
+    -2. If the opponent's NCS is below 0, the play defects; otherwise,
+    they cooperate.
+
+    The player's memories of the opponent's decisions have a random chance to be
+    altered (i.e., a C decision becomes D or vice versa; default probability
+    is 0.03) or deleted (default probability is 0.1).
+
+    It's necessary to specify EXACT name of the starting strategy if changing
```

Comment:
no need for the capitals and should instead read '... specify the exact  name ...'